Source/Other/Data/SPConstants.m

```diff
@@ -456,6 +456,7 @@ - (SPTableViewType)tableViewTypeEnumFromString{
 NSString *SPBundleShellVariableSelectedText                 = @"SP_SELECTED_TEXT";
 NSString *SPBundleShellVariableSelectedTextRange            = @"SP_SELECTED_TEXT_RANGE";
 NSString *SPBundleShellVariableUsedQueryForTable            = @"SP_USED_QUERY_FOR_TABLE";
+NSString *SPMigratedQueriesFromPrefs            			= @"SPMigratedQueriesFromPrefs";
```

Comment:
Maybe this is git being weird but looks like spacing is funky before the =?

Reply:
will fix